apps/central-scan/backend/src/interpret_sheet.test.ts

```diff
@@ -0,0 +1,261 @@
+              Object {
+                "id": "vincent-van-gogh",
+                "name": "Vincent Van Gogh",
+                "partyIds": Array [
+                  "1",
+                ],
+              },
+            ],
+            "chief-of-police": Array [
+              Object {
+                "id": "natalie-portman",
+                "name": "Natalie Portman",
+                "partyIds": Array [
+                  "0",
+                ],
+              },
+            ],
+            "city-council": Array [
+              Object {
+                "id": "marie-curie",
+                "name": "Marie Curie",
+                "partyIds": Array [
+                  "0",
+                ],
+              },
+              Object {
+                "id": "indiana-jones",
+                "name": "Indiana Jones",
+                "partyIds": Array [
+                  "1",
+                ],
+              },
+              Object {
+                "id": "mona-lisa",
+                "name": "Mona Lisa",
+                "partyIds": Array [
+                  "3",
+                ],
+              },
+              Object {
+                "id": "jackie-chan",
+                "name": "Jackie Chan",
+                "partyIds": Array [
+                  "3",
+                ],
+              },
+            ],
+            "controller": Array [
+              Object {
+                "id": "winston-churchill",
+                "name": "Winston Churchill",
+                "partyIds": Array [
+                  "0",
+                ],
+              },
+            ],
+            "mayor": Array [
+              Object {
+                "id": "sherlock-holmes",
+                "name": "Sherlock Holmes",
+                "partyIds": Array [
+                  "0",
+                ],
+              },
+            ],
+            "parks-and-recreation-director": Array [
+              Object {
+                "id": "charles-darwin",
+                "name": "Charles Darwin",
+                "partyIds": Array [
+                  "0",
+                ],
+              },
+            ],
+            "public-works-director": Array [
+              Object {
+                "id": "benjamin-franklin",
+                "name": "Benjamin Franklin",
+                "partyIds": Array [
+                  "0",
+                ],
+              },
+            ],
+          },
+        },
+      },
+      Object {
+        "interpretation": Object {
+          "type": "BlankPage",
+        },
+      },
+    ]
+  `);
+});
+
+test('properly detects test ballot in live mode', async () => {
+  const ballotImagePath =
+    electionFamousNames2021Fixtures.machineMarkedBallotPage1.asFilePath();
+  const interpretationResult = await interpretSheet(
+    {
+      electionDefinition: electionFamousNames2021Fixtures.electionDefinition,
+      precinctSelection: ALL_PRECINCTS_SELECTION,
+      testMode: false, // this is the test mode
```

Comment:
I know this is just copied from a prior file, but I think this comment is a bit confusing - can probably remove